### PR TITLE
Clear Creek and Merced Edits 

### PR DESCRIPTION
**Pull request by ErinCain**
Merced finished except steelhead mark gard problem. Clear needs data fixes

### CODE REVIEW

**Comment by ErinCain** [comment]
I cannot seem to create a new pull request for the next documentation that I finished so I am commenting here to say that this now includes Calaveras which is complete. 

Close #25 (calaveras issue).

**Comment by ErinCain** [comment]
Includes edits to mokelumne

**Comment by ErinCain** [comment]
Close #23 Finished bear including Mark's Edits to description.

File: data-raw/instream/merced/merced_river.rmd [hunk 1]
```diff
@@ -13,14 +13,14 @@ library(readxl)
...
 ## Future Data Improvements
-none
-
+none planned
+ 
 ## Instream Spawning and Rearing Habitat
 
 **Data Source:**
-[FWS 1996 and MID 2013](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/FWS1996_AmericanRiverSpawningFinalReport.pdf)
+[Merced ID 2013](https://cvpiahabitat-r-package.s3-us-west-2.amazonaws.com/cvpia-sit-model-inputs/MID2013_MercedIFIMStudy.pdf) (pg 67-75)
```
Comment (thesadie) [inline data-raw/instream/merced/merced_river.rmd]: Either MID or Merced Irrigation District

File: data-raw/instream/merced/merced_river.rmd [hunk 2]
```diff
@@ -13,14 +13,14 @@ library(readxl)
...
+Instream spawning and rearing habitat for Fall Run Chinook Salmon and Steelhead in the Merced River is based on data from a 2011 and 2012 Merced Irrigation District (Merced ID) flow-habitat study in the 19.2 mile-long section of the Merced River from Crocker-Huffman Diversion Dam at river mile (RM) 52.01 downstream to Shaffer Bridge at RM 32.8. Habitat data from this study is summarized on pages 67 - 75 of the Merced ID 2013 study. The instream spawning and rearing habitat data was provided to us by [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service in a spreadsheet (see '~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx') that creates an average of the WUA values for the three study reaches weighted by the length of each study reach. Mark Gard instructed us to use A18:E48 of the 'Merced' tab within his 'IFIMWUA.xlsx'.
```
Comment (thesadie) [inline data-raw/instream/merced/merced_river.rmd]: change Merced ID to MID

File: data-raw/instream/mokelumne/mokelumne_river.Rmd [hunk 3]
```diff
@@ -22,13 +22,13 @@ library(scales)
...
 
-### Spawning WUA
+Instream spawning and rearing habitat for Fall Run Chinook Salmon in the Mokelumne River is based on data from USFWS's 1998 instream flow evaluation and EBMUD's 2016 instream habitat evaluation. The EBMUD instream spawning and rearing habitat data was provided in a spreadsheet (see '~/cvpiaHabitat/data-raw/instream/mokelumne/LMR_habitat__s_v2mw_to_MT.xlsx') Together, these evaluations developed relationships between flow and suitable spawning and rearing habitat for reaches spanning approximately 35 miles of the Mokelumne River. 
 
-EBMUD provided a spawning habitat flow:area relationship covering the entire river. For each flow, the area was converted to an area per unit length by dividing the total spawning area by the length of the spawning reach mapped by the SIT.
```
Comment (thesadie) [inline data-raw/instream/mokelumne/mokelumne_river.Rmd]: not sure if the EBMUD provided spawning habitat is based on 1998 study

File: data-raw/instream/mokelumne/mokelumne_river.Rmd [hunk 4]
```diff
@@ -129,57 +111,94 @@ rearing <- r1 %>%
...
+
+
+```
+
+```{r, echo = FALSE}
+knitr::kable(align = 'c' , head(mokelumne_river_instream, 5))
+```
+
+
+###### ...with 47 more rows.
```
Comment (thesadie) [inline data-raw/instream/mokelumne/mokelumne_river.Rmd]: change to italics

File: data-raw/instream/mokelumne/mokelumne_river.Rmd [hunk 5]
```diff
@@ -129,57 +111,94 @@ rearing <- r1 %>%
...
+  ggplot(aes(x = flow_cfs , y = wua, color = lifestage)) +
+  geom_line() +
+  labs(x = 'flow (cfs)', y = 'WUA (sqft/1000ft)') + theme_minimal() +
+  scale_color_manual(values = c('#d95f02','#7570b3'))
 ```
 
+
+
+```{r, include = FALSE, echo=FALSE}
 ## Floodplain
```
Comment (thesadie) [inline data-raw/instream/mokelumne/mokelumne_river.Rmd]: Add back in all this floodplain stuff

File: data-raw/instream/stanislaus/stainislaus_river.rmd [hunk 6]
```diff
@@ -15,39 +15,42 @@ library(scales)
...
+Instream spawning and rearing habitat for Fall Run Chinook Salmon and Steelhead in the Stanislaus River is based on data from Aceituno 1993 and FWS 2012. The Aceituno 1993 study uses the PHABSIM model which was developed in 1989 to look at the relationship between physical habitat availability and streamflow for Chinook Salmon Spawning in approximately 24 miles of the Stanislaus.THe original spawning data is on pg 19 of the report. THe rearing data comes from the 2012 FWS CVPIA Annual progress report.The FWS used four sites on the river and conducted 2-D hydraulic modeling of all the sites a graph representing the rearing totals for all the sites is on pg 6 of the report. The data was provided to us by [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service in a spreadsheet (see '~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx'). Mark Gard instructed us to use I4:N60 of the 'Stanislaus' tab within his 'IFIMWUA.xlsx' These cells represent the sums of the rearing habitat in the canyon segment, knights ferry to orange blossom, orange blossom to jacomb myers, and jacob myers to san joaquin and then converted from sqft to sqft/1000ft. Spawning is modeled
```
Comment (thesadie) [inline data-raw/instream/stanislaus/stainislaus_river.rmd]: fix the in 'THe original spawning' there are two cases of 'THe'
also there are no spaces before a few sentences, plz fix

File: data-raw/instream/stanislaus/stainislaus_river.rmd [hunk 7]
```diff
@@ -15,39 +15,42 @@ library(scales)
...
+as one 50 mi segment (REPORT SAYS @$ MILES***THIS MIGHT BE THE REASON FOR THE DATA CONFUSION.
```
Comment (thesadie) [inline data-raw/instream/stanislaus/stainislaus_river.rmd]: did you figure out the units issue?

File: data-raw/instream/stanislaus/stainislaus_river.rmd [hunk 8]
```diff
@@ -15,39 +15,42 @@ library(scales)
...
+stanislaus_river_instream <- stanislaus_raw %>% 
+   mutate(watershed = 'Stanislaus River') %>% 
+   select(flow_cfs, FR_spawn_wua, FR_fry_wua, FR_juv_wua, ST_fry_wua, ST_juv_wua, watershed)
+ 
+knitr::kable( align = 'c', head(stanislaus_river_instream, 5))
+ # devtools::use_data(stanislaus_river_instream, overwrite = TRUE)
+ 
 ```
 
+###### ...with 52 more rows.
```
Comment (thesadie) [inline data-raw/instream/stanislaus/stainislaus_river.rmd]: change to italics

File: data-raw/instream/bear_river/bear_river.rmd [hunk 9]
```diff
@@ -13,14 +13,14 @@ library(readxl)
...
 ## Future Data Improvements
-none
+None Planned
 
 ## Instream Spawning and Rearing Habitat
 
 **Data Source:**
 [Holton 1985](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/Holton1985_BearRiverIFIM.PDF) (pg 21-22)
 
-Instream spawning and rearing habitat for Fall Run Chinook Salmon in the Bear River is based on data from a personal project by Booker Holton in partnership with the California Department of Fish and Game. Holton used the Instream Flow Incremental Methodology to complete his study and collected data from two representative stations below the diversion dam on Bear River. The WUA graphs of spawning and juvenile Salmon are provided on pages 21 and 22. The instream spawning and rearing habitat data was provided to us by [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service in a spreadsheet (see '~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx').Mark Gard instructed us to use A1:D5 of the 'BearR' tab within his 'IFIMWUA.xlsx'.
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/instream/bear_river/bear_river.rmd]: done

File: data-raw/instream/bear_river/bear_river.rmd [hunk 10]
```diff
@@ -1,7 +1,7 @@
 ---
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/instream/bear_river/bear_river.rmd]: got rid of steelheed but couldn't find the .Rhiostory?

File: data-raw/instream/calaveras/calaveras_river.rmd [hunk 11]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Calaveras River"
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/instream/calaveras/calaveras_river.rmd]: done

File: data-raw/instream/clear/clear_creek.rmd [hunk 12]
```diff
@@ -14,105 +14,127 @@ library(cvpiaHabitat)
...
 
 ## Instream Spawning and Rearing Habiat
 
-[????]
-**Data Source:** FWS 2007-2013 TODO
-<https://flowwest.github.io/cvpiaHabitat/reference/clear_creek_instream.html>
+**Data Source:**     
+[FWS 2007](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Whiskeytown%20to%20Clear%20Creek%20Br%20spawning%20final%20report.pdf) (pgs 113-117)    
+[FWS Rearing 2011](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Whiskeytown%20to%20Clear%20Creek%20Br%20rearing%20final%20report.pdf) (pgs 144-157)    
+[FWS Spawning 2011](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Lower%20Spawning%20Final%20Report.pdf)(pgs 107-109)
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/instream/clear/clear_creek.rmd]: done

File: data-raw/instream/clear/clear_creek.rmd [hunk 13]
```diff
@@ -14,105 +14,127 @@ library(cvpiaHabitat)
...
 ## Instream Spawning and Rearing Habiat
 
-[????]
-**Data Source:** FWS 2007-2013 TODO
-<https://flowwest.github.io/cvpiaHabitat/reference/clear_creek_instream.html>
+**Data Source:**     
+[FWS 2007](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Whiskeytown%20to%20Clear%20Creek%20Br%20spawning%20final%20report.pdf) (pgs 113-117)    
+[FWS Rearing 2011](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Whiskeytown%20to%20Clear%20Creek%20Br%20rearing%20final%20report.pdf) (pgs 144-157)    
+[FWS Spawning 2011](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Lower%20Spawning%20Final%20Report.pdf)(pgs 107-109)    
+[FWS 2013](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Lower%20Rearing%20Final%20Report-1.pdf)(pgs 150-160)
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/instream/clear/clear_creek.rmd]: done

File: data-raw/instream/mokelumne/mokelumne_river.Rmd (hunk 4 above)
Comment (ErinCain) [reply data-raw/instream/mokelumne/mokelumne_river.Rmd]: done

File: data-raw/instream/mokelumne/mokelumne_river.Rmd (hunk 5 above)
Comment (ErinCain) [reply data-raw/instream/mokelumne/mokelumne_river.Rmd]: done

File: data-raw/instream/stanislaus/stainislaus_river.rmd (hunk 6 above)
Comment (ErinCain) [reply data-raw/instream/stanislaus/stainislaus_river.rmd]: done

File: data-raw/instream/stanislaus/stainislaus_river.rmd (hunk 7 above)
Comment (ErinCain) [reply data-raw/instream/stanislaus/stainislaus_river.rmd]: Still need to check with Mark/Mark Gard to figure out issue. Commented in project issue to keep track of.

File: data-raw/instream/stanislaus/stainislaus_river.rmd (hunk 8 above)
Comment (ErinCain) [reply data-raw/instream/stanislaus/stainislaus_river.rmd]: done

File: data-raw/instream/cosumnes/cosumnes_river.Rmd [hunk 14]
```diff
@@ -15,70 +15,74 @@ library(scales)
...
-#                                   watershed = 'Cosumnes River')
-# 
-# cosumnes_river_instream
+[Calaveras, FISHBIO Environmental and Thomas R. Payne & Associates 2009](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/FishBio_Payne2009_CalaverasInstreamFlowStudy.pdf)    
+[Mokelumne, CDFW 1991](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/Lower+Mokelumne+RIver+Fisheries+Management+Plan%2C+CDFG+1991.pdf)
+
+No watershed specific salmonid habitat data was available for Consumnes River. A regional weighted usable area (WUA) relationship with flow was derived for Consumnes River by averaging the WUA values on Calaveras River and Mokelumne River. The geomorphic and hydrologic conditions in Consumnes River (watershed area = X; active channel width = Y; 2-year flow = Z) are similar to those on Calaveras River (watershed area = X; active channel width = Y; 2-year flow = Z), Mokelumne River (watershed area = X; active channel width = Y; 2-year flow = Z). The regional WUA relationships for Calaveras River were multiplied by the length of spawning and rearing extents mapped by the Science Integration Team (SIT).
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/instream/cosumnes/cosumnes_river.Rmd]: note is in issue

File: data-raw/instream/cosumnes/cosumnes_river.Rmd [hunk 15]
```diff
@@ -15,70 +15,74 @@ library(scales)
...
+   select(Watershed, Region, FR_fry, FR_juv)
+ 
+ flows <- seq(100, 1000, by = 100)
+ 
+ #spawn
+ cal_st_spwn <- approx(calaveras_river_instream$flow_cfs, 
+                       calaveras_river_instream$ST_spawn_wua, 
+                       rule = 2, xout = flows)$y
+
+ mok_fr_spwn <- approx(mokelumne_river_instream$flow_cfs,
```
[the remark this answers is not in the document]
Comment (ErinCain) [reply data-raw/instream/cosumnes/cosumnes_river.Rmd]: Looks like it is updated to reflect recent moke and calaveras data.

File: data-raw/instream/mokelumne/mokelumne_river.Rmd (hunk 3 above)
Comment (ErinCain) [reply data-raw/instream/mokelumne/mokelumne_river.Rmd]: I don't have another source that is showing up. Will look for the correct source. I left a comment on mokelumne issue.

**Pull request merged**